events.go

```diff
@@ -26,7 +26,7 @@ type Event interface {
 }
 
 type BaseEvent struct {
-	CreationDate uint64 `json:"creationDate"`
+	CreationDate float64 `json:"creationDate"`
```

Comment:
When kept as uint64 this can get marshaled to scientific notation, which then is not parsable as a uint64.